src/Controller/Organization/MissionType/MissionTypeDeleteController.php

```diff
@@ -34,7 +34,7 @@ public function __invoke(MissionType $missionType): RedirectResponse
         $this->entityManager->remove($missionType);
         $this->entityManager->flush();
 
-        $this->addFlash('success', $this->translator->trans('organization.mission_type.delete_success_message'));
+        $this->addFlash('success', $this->translator->trans('organization.missionType.deleteSuccessMessage'));
```

Comment:
I think you can do `$this->addFlash('success', 'organization.missionType.deleteSuccessMessage');`

Reply:
And if the flash message is not translated, it would be better to do it in the template